docs/containers-registries.d.5.md

```diff
@@ -77,6 +77,10 @@ described above.  The configuration section is a YAML mapping, with the followin
    This key is optional; if it is missing, no signature storage is defined (no signatures
    are download along with images, adding new signatures is possible only if `sigstore-staging` is defined).
 
+The default `sigstore-staging` is `/var/lib/containers/sigstore` from `/etc/containers/registries.d/default.yaml`.
+If the user has no privilege to write the signature to the default sigstore-staging, the YAML format files
+under `$HOME/.config/containers/registries.d` can be configured, otherwise, `$HOME/.local/share/containers/sigstore` will be used.
```

Comment:
All of this should be in the ”Registries, Scopes and Search Order” section, maybe in an extra named subsection.